semeio/communication/semeio_script.py

```diff
@@ -83,10 +83,25 @@ def _get_output_dir(self):
         return os.path.join(
             base_dir,
             "reports",
+            Path(res_config.user_config_file).stem,
             sub_dir,
             type(self).__name__,
         )
 
     @property
     def reporter(self):
         return self._reporter
+
+    @property
+    def _reports_dir(self):
```

Comment:
How is this working out when the method and the private variable shares the same name?